src/Surface.zig

```diff
@@ -4975,6 +4975,15 @@ pub fn performBindingAction(self: *Surface, action: input.Binding.Action) !bool
             },
         ),
 
+        .goto_window => |direction| return try self.rt_app.performAction(
+            .{ .surface = self },
+            .goto_window,
+            switch (direction) {
+                .next => apprt.action.GotoWindow.next,
+                .previous => apprt.action.GotoWindow.previous,
```

Comment:
Decl literals should just work here, no?
```suggestion
                .next => .next,
                .previous => .previous,
```